### PR TITLE
OpenCollective post

### PR DESCRIPTION
**Pull request by Razican**
This adds the post for the OpenCollective organisation. Still in draft.

### CODE REVIEW

File: _posts/2022-03-01-boa-open-collective.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Boa JS engine enables financial contributions"
+author: Boa Developers
+---
+
+In the past months, due to the big and steady progress on Boa, we have received multiple requests to contribute with Boa
+financially. From now on, anyone can contribute with either a recurring donationk, or a one-time contribution in our
```
Comment (jedel1043) [inline _posts/2022-03-01-boa-open-collective.md]:
```suggestion
financially. From now on, anyone can contribute with either a recurring donation, or a one-time contribution in our
```

File: _posts/2022-03-01-boa-open-collective.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+[OpenCollective page](https://opencollective.com/boa).
+
+# What does this mean for Boa?
+
+Being able to receive financial contributions won't directly show a change in Boa development, but depending on how much
+funding we get, we are already planning next steps. Having more motivated contributors would be the first step. We all
+want to spend more of our free time in boa, and receiving a financial compensation for it is definitely makin the team
```
Comment (jedel1043) [inline _posts/2022-03-01-boa-open-collective.md]:
```suggestion
want to spend more of our free time in boa, and receiving a financial compensation for it is definitely making the team
```

File: _posts/2022-03-01-boa-open-collective.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+members eager to continue contributing.
+
+But that is not our only idea. We would like to create a reproducible benchmarking platform. Currently, we just use GitHub
+actions for benchmarking. While this is useful to get some general ideas on how do changes affect to the performance of the
+engine, but they have a high variability (up to 10%). This means that we can definitely see changes if they are big, but it's
```
Comment (jedel1043) [inline _posts/2022-03-01-boa-open-collective.md]:
```suggestion
actions for benchmarking, which is useful to get some general ideas on how some changes affect the performance of the
engine. However, they have a high variability (up to 10%), meaning that we can definitely see changes if they are big, but it's
```

**Review by jedel1043 — CHANGES_REQUESTED** [review]

File: _posts/2022-03-01-boa-open-collective.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Boa JS engine enables financial contributions"
+author: Boa Developers
+---
+
+In the past months, due to the big and steady progress on Boa, we have received multiple requests to contribute with Boa
```
Comment (RageKnify) [inline _posts/2022-03-01-boa-open-collective.md]:
```suggestion
In the past months, due to the big and steady progress on Boa, we have received multiple requests to contribute to Boa
```

**Review by RageKnify — APPROVED** [review]
I like jedel's suggestions, don't have anything else to add.

**Review by HalidOdat — APPROVED** [review]
This looks perfect to me! :)

**Review by jedel1043 — APPROVED** [review]

**Comment by Razican** [comment]
Merged with #20.

**Pull request closed without merging**